.coveragerc

```diff
@@ -3,6 +3,7 @@ skip_covered = True
 show_missing = True
 fail_under = 75
 exclude_lines =
-    pragma: no cover
-    if __name__ == .__main__.:
     def __repr__
+    if __name__ == .__main__.:
```

Comment:
Why did this line move? Did you sort this alphabetically manually or accidentally with some tool? Anyway, this line change is totally unnecessary especially since the file is going away in favour of pyproject.toml later in this series.